### PR TITLE
add Header + mobile menu

### PR DESCRIPTION
**Pull request by Sysqwerty**

### CODE REVIEW

File: src/partials/index_scripts.html [hunk 1]
```diff
@@ -1 +1,6 @@
-<script src="./js/index.js" type="module"></script>
\ No newline at end of file
+<script src="./js/index.js" type="module"></script>
+<script defer src="https://cdnjs.cloudflare.com/ajax/libs/body-scroll-lock/3.1.5/bodyScrollLock.min.js"
```
Comment (olexandrd) [inline src/partials/index_scripts.html]: lets setup body-scroll-lock using npm

Comment (olexandrd) [reply src/partials/index_scripts.html]: npm i body-scroll-lock
https://www.npmjs.com/package/body-scroll-lock#reactes6 for example

Comment (Sysqwerty) [reply src/partials/index_scripts.html]: Done. Replaced with npm realization.

File: src/css/mobile_menu.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+.menu-toggle.menu-close-btn {
+  align-self: flex-end;
+}
+
+.menu-close-icon {
+  stroke: var(--light);
+  transition: stroke var(--transition);
+}
+
+/* .menu-close-btn:hover .menu-close-icon,
```
Comment (kazamov) [inline src/css/mobile_menu.css]: Remove commented code

Comment (Sysqwerty) [reply src/css/mobile_menu.css]: Removed

**Review by kazamov — APPROVED** [review]

**Pull request merged**